Author: Martin Pitt <mpitt@debian.org>
Description: Use version specific installation directories so that several major versions can be installed in parallel.
Forwarded: No, Debian specific packaging with postgresql-common

 * Install lib files into /usr/lib/postgresql/<version>/lib/
 * Install server related header files into /usr/include/postgresql/<version>/server/

Bug-Debian: http://bugs.debian.org/462037

--- a/src/Makefile.global.in
+++ b/src/Makefile.global.in
@@ -119,7 +119,7 @@ libdir := @libdir@
 pkglibdir = $(libdir)
 ifeq "$(findstring pgsql, $(pkglibdir))" ""
 ifeq "$(findstring postgres, $(pkglibdir))" ""
-override pkglibdir := $(pkglibdir)/postgresql
+override pkglibdir := /usr/lib/postgresql/@PG_MAJORVERSION@/lib
 endif
 endif
 
@@ -167,7 +167,7 @@ endif # PGXS
 
 # These derived path variables aren't separately configurable.
 
-includedir_server = $(pkgincludedir)/server
+includedir_server = $(pkgincludedir)/@PG_MAJORVERSION@/server
 includedir_internal = $(pkgincludedir)/internal
 pgxsdir = $(pkglibdir)/pgxs
 bitcodedir = $(pkglibdir)/bitcode
